node/clients/LMClient.ts

```diff
@@ -24,6 +26,24 @@ export default class LMClient extends ExternalClient {
     })
   }
 
+  public checkUserAdminPermission = async (
+    account: string,
+    userEmail: string,
+    resourceCode: string
+  ) => {
+    const productCode = B2B_LM_PRODUCT_CODE // resource name on lincense manager = B2B
```

Comment:
the actual name of this resource is `Buyer Organization` actually (I gave you the wrong info before). So lets update the comment here
```suggestion
    const productCode = B2B_LM_PRODUCT_CODE // resource name on lincense manager = Buyer Organization
```